src/components/Restoration/ExtendTimeLimit.vue

```diff
@@ -92,7 +93,6 @@ export default class ExtendTimeLimit extends Vue {
     return this.getStateFilingRestoration?.approvalType
   }
 
-  /** The court order draft file number. */
   get courtOrderNumberText (): string {
```

Comment:
If this method is now obsolete, might as well delete it.

Reply:
Removed